### PR TITLE
Remove d3 dependency, Code refactoring

### PR DESCRIPTION
**Pull request by DimaNaumov**

### CODE REVIEW

File: src/globemap.ts [hunk 1]
```diff
@@ -906,11 +906,10 @@ module powerbi.extensibility.visual {
             }
             this.layout.viewport = options.viewport;
             this.root.css(this.layout.viewportIn);
-            this.zoomContainer.style({
-                "display": this.layout.viewportIn.height > GlobeMap.ZoomControlSettings.height
+            this.zoomContainer.setAttribute("style",
```
Comment (ignatvilesov) [inline src/globemap.ts]: Does `setAttribute` work in IE 11?

File: src/globemap.ts [hunk 2]
```diff
@@ -1292,18 +1291,13 @@ module powerbi.extensibility.visual {
...
-                    callback(<SVGElement> (d3.event as MouseEvent).currentTarget);
-                }
-            }
-            this.zoomContainer = d3.select(controlContainer);
-            this.zoomContainer
-                .selectAll("g")
-                .on("mousedown", () => onMouseDown(
-                    (element: SVGElement) => {
-                        const controlType = element.classList.toString().split(" ").filter(className => className.search("js-") !== -1)[0];
+            this.zoomContainer = document.querySelector("div.controls-container");
```
Comment (ignatvilesov) [inline src/globemap.ts]: zoomContainer is actually controlContainer in the previous version

Comment (ignatvilesov) [reply src/globemap.ts]: Can we use controlContainer instead?

File: src/globemap.ts [hunk 3]
```diff
@@ -1292,18 +1291,13 @@ module powerbi.extensibility.visual {
...
+            let allG = this.zoomContainer.querySelectorAll("g");
+            for (let i = 0; i < allG.length; ++i) {
+                allG[i].onmousedown = (event) => {
+                    event.stopPropagation();
+                    if (event.button === 0) {
+                        const controlType = (event.currentTarget as HTMLHtmlElement).classList.toString().split(" ").filter(className => className.search("js-") !== -1)[0];
```
Comment (ignatvilesov) [inline src/globemap.ts]: Please check it with IE 11

File: package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "powerbi-visuals-globemap",
   "description": "GlobeMap",
-  "version": "2.2.2",
+  "version": "2.3.0",
```
Comment (ignatvilesov) [inline package.json]: Do we still need `"@types/d3": "3.5.36",`?

File: src/globemap.ts (hunk 1 above)
Comment (DimaNaumov) [reply src/globemap.ts]: https://www.w3schools.com/jsref/met_element_setattribute.asp

Comment (DimaNaumov) [reply src/globemap.ts]: How i see, it is work in IE 8 and higher

File: package.json (hunk 4 above)
Comment (DimaNaumov) [reply package.json]: Pbi utils use @types/d3

File: src/globemap.ts (hunk 2 above)
Comment (DimaNaumov) [reply src/globemap.ts]: Fixed

File: src/globemap.ts (hunk 3 above)
Comment (DimaNaumov) [reply src/globemap.ts]: Checked, all is ok

**Pull request merged**